Jenkinsfile

```
@@ -38,5 +39,5 @@ withCredentials([string(credentialsId: 'atlas_appcenter_integration_token', vari
                                 ],
                           ]
 
-    buildGradlePlugin plaforms: ['osx', 'windows', 'linux'], coverallsToken: coveralls_token, testEnvironment: testEnvironment
+    buildGradlePlugin plaforms: ['osx', 'windows', 'linux'], coverallsToken: coveralls_token, sonarToken: sonar_token, testEnvironment: testEnvironment
```

I know its not related to your change but could we try to fix the stupid spelling error I made like 3 years ago?
```suggestion
    buildGradlePlugin platforms: ['osx', 'windows', 'linux'], coverallsToken: coveralls_token, sonarToken: sonar_token, testEnvironment: testEnvironment
```